keyboards/keychron/k8/rgb/keymaps/via_iso/config.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 Harrison Chan (Xelus)
+ * Copyright 2022 Thomas Bowman Mørch (tbowmo)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// Enable RGB Matrix
+#define VIA_QMK_RGBLIGHT_ENABLE
+#define DYNAMIC_KEYMAP_LAYER_COUNT 4
```

Comment:
safe to omit, see https://github.com/SonixQMK/qmk_firmware/blob/sn32_develop/quantum/dynamic_keymap.c#L36-L38

Reply:
removed this